libs/application/templates/parental-leave/src/lib/ParentalLeaveForm.ts

```diff
@@ -388,17 +388,30 @@ export const ParentalLeaveForm: Form = buildForm({
                 'You can choose to start on the date of birth, or on a specific date. Please note, that your rights end 18 months after the date of birth.',
               placeholder: 'Pick the start date',
             }),
-            buildCustomField(
-              {
-                id: 'endDate',
-                name: m.duration,
-                description: m.durationDescription,
-                component: 'ParentalLeaveDuration',
-              },
-              {
-                showTimeline: true,
-              },
-            ),
+            buildMultiField({
+              id: 'endDate',
+              name: 'Please pick the end date',
+              description:
+                'You can choose to end the parental leave no later than 18 months after the date of birth.',
+              children: [
+                buildDateField({
+                  id: 'endDate',
+                  name: 'End date',
+                  placeholder: 'Pick the end date',
+                }),
+              ],
+            }),
+            // buildCustomField(
```

Comment:
Should we remove commented out part?

Reply:
This is subject to change when I combine start and end date pickers into one screen. I will trim it then